src/storage-types/src/connections/aws.rs

```diff
@@ -223,14 +221,11 @@ impl AwsAssumeRole {
         // name here, so that we can identify the specific environment and
         // connection ID that initiated the session in our internal CloudTrail
         // logs. This session isn't visible to the end user.
-        let mut jump_credentials = AssumeRoleProvider::builder(aws_connection_role_arn)
-            .session_name(default_session_name.clone());
-
-        if let Some(region) = region.clone() {
-            jump_credentials = jump_credentials.region(region);
-        }
```

Comment:
@guswynn I validated on staging that all works as intended, even without explicitly setting the region here.

Reply:
helllll ya! so much cleaner!